src/rust/ide/lib/analytics/src/remote_log.rs

```diff
@@ -9,27 +9,44 @@ pub use wasm_bindgen::prelude::*;
 
 
 #[wasm_bindgen(inline_js="
-export function _remote_log(msg, data) {
-    if (window !== undefined && window.enso !== undefined && window.enso.remote_log !== undefined) {
-        window.enso.remote_log(msg,{data:data})
+export function _remote_log(msg, value) {
+    if (window !== undefined && window.enso !== undefined && window.enso.remoteLog !== undefined) {
+        try {
+            if (value === undefined && value !== null) {
+                window.enso.remoteLog(msg)
+            } else {
+                window.enso.remoteLog(msg,value)
+            }
+        } catch (error) {
+            console.error(\"Error while logging message. \" + error );
+        }
     } else {
         console.warn(\"Failed to send log message.\")
     }
```

Comment:
I don't like one thing in this wrapper a lot. Imagine that we will refactor the code and rename the function `window.enso.remoteLog` to `window.luna.remoteLog` (or just anything else). This wrapper will silently fail and discovery during development why logs are not working anymore could take days. IMO, this should just be:

```js
try {
    window.enso.remoteLog(msg)
} catch (error) {
    console.error(\"Error while logging message. \" + error );
}
```

If `window.enso.remoteLog` is not present, it means that something is deeply broken and we should know about it.